lectures/markov_asset.md

```diff
@@ -1017,54 +991,52 @@ def sv_pd_ratio_jax_multi(sv_model, shapes):
     # Compute v
     ones_array = np.ones((I, J, K))
     # Set up the operator g -> g
-    H = lambda g: H_operator(g, sv_model, shapes)
+    H = lambda g: H(g, sv_model, shapes)
     # Set up the operator g -> (I - H) g
     J = lambda g: g - H(g)
     # Solve v = (I - H)^{-1} H 1
-    H1 = H_operator(ones_array, sv_model, shapes)
+    H1 = H(ones_array, sv_model, shapes)
```

Comment:
Looks like H1 is unnecessary here and in the next line we should be calling the solver on H rather than H1.